### PR TITLE
feat(website): Update node troubleshooting for Windows users.

### PR DESCRIPTION
**Pull request by ikaika1**
Problem: When trying to run a node on Windows, an error occurs - simple-taiko-node-taiko_client_prover_relayer-1 | /bin/sh:/script/start-prover-relayer.sh:not found.

Fix: Updated steps in docs so that no errors occur when running a node on Windows and added leppaludi's nice suggested fix in the troubleshooting section of the docs.

### CODE REVIEW

**Review by davidtaikocha — DISMISSED** [review]

File: packages/website/pages/docs/guides/run-a-node.mdx [hunk 1]
```diff
@@ -19,10 +19,31 @@ This guide will walk you through the process of operating a Taiko node via [simp
...
+  </Tab>
+  <Tab>
+  ```sh
+  git clone https://github.com/taikoxyz/simple-taiko-node.git
+  cd simple-taiko-node
+  ```
+  </Tab>
+  <Tab>
+  ```sh
+  git config --global core.autocrlf false.
```
Comment (d1onys1us) [inline packages/website/pages/docs/guides/run-a-node.mdx:40]:
```suggestion
  git config --global core.autocrlf false
```
i think we need this right?

File: packages/website/pages/docs/guides/run-a-node.mdx [hunk 2]
```diff
@@ -19,10 +19,31 @@ This guide will walk you through the process of operating a Taiko node via [simp
...
+  git clone https://github.com/taikoxyz/simple-taiko-node.git
+  cd simple-taiko-node
+  ```
+  </Tab>
+  <Tab>
+  ```sh
+  git clone https://github.com/taikoxyz/simple-taiko-node.git
+  cd simple-taiko-node
+  ```
+  </Tab>
```
Comment (d1onys1us) [inline packages/website/pages/docs/guides/run-a-node.mdx:37]:
```suggestion
<Tabs items={["Mac/Linux", "Windows"]} defaultIndex="0">
  <Tab>
  ```sh
  git clone https://github.com/taikoxyz/simple-taiko-node.git
  cd simple-taiko-node
  ```
  </Tab>
```
i would recommend this instead for minimum viable documentation and to not duplicate.

**Comment by d1onys1us** [comment]
hi @ikaika1 i left some small comments. otherwise looks great, thanks!

File: packages/website/pages/docs/guides/run-a-node.mdx (hunk 1 above)
Comment (ikaika1) [reply packages/website/pages/docs/guides/run-a-node.mdx:40]: I apologize for the mistake. Thank you for making the correction.

File: packages/website/pages/docs/guides/run-a-node.mdx (hunk 2 above)
Comment (ikaika1) [reply packages/website/pages/docs/guides/run-a-node.mdx:37]: Thank you very much! It's true that ["Mac/Linux", "Windows"] is simpler and easier to understand!

**Review by dionysuzx — DISMISSED** [review]

**Comment by MarcusWentz** [comment]
I see the following issues with tabs

**Review by MarcusWentz — CHANGES_REQUESTED** [review]
Please fix
https://github.com/taikoxyz/taiko-mono/pull/14032#issuecomment-1604967806
I think you just need a closing Tab tag.

**Review by 2manslkh — APPROVED** [review]

**Review by MarcusWentz — APPROVED** [review]

**Pull request merged**